### PR TITLE
Hakan

### PR DESCRIPTION
**Pull request by hakankara75**

### CODE REVIEW

File: TestOutput/failed_scenario.txt [hunk 1]
```diff
@@ -1 +1 @@
-file:src/test/resources/features/hakan.feature:12
+file:src/test/resources/features/hakan2.feature:23:24:25
```
Comment (HakanKara75) [inline TestOutput/failed_scenario.txt:1]: v

File: TestOutput/failed_scenario.txt
Comment (hakankara75) [inline TestOutput/failed_scenario.txt:1]: a

File: configuration.properties
Comment (hakankara75) [inline configuration.properties:1]: a

File: src/test/java/techproed/pages/CokSatanKitaplar_Edebiyat_Hakan.java
Comment (hakankara75) [inline src/test/java/techproed/pages/CokSatanKitaplar_Edebiyat_Hakan.java:1]: a

File: src/test/java/techproed/pages/Login.java
Comment (hakankara75) [inline src/test/java/techproed/pages/Login.java:1]: a

**Review by hakankara75 — COMMENTED** [review]
t

File: TestOutput/failed_scenario.txt
Comment (hakankara75) [inline TestOutput/failed_scenario.txt:1]: fgsdfbfdsbdefhgrynjthmtymeutekftr

**Review by hakankara75 — COMMENTED** [review]
gfhedhbathetdbhant

**Pull request closed without merging**